pydicom/encaps.py

```diff
@@ -22,7 +22,7 @@
 # If a single frame, it may be 0 length/no value,
 # or it may have a single pointer (0).
 
-from pydicom.config import logger
+import pydicom.config
```

Comment:
Yes, I agree.  I had to change it because we now have a circular reference with pydicom.config.  It now tries to figure out what image handlers are available by trying to import them.  Those image handlers now include pydicom.encaps since I added enhanced MR support.  
Circular references aren't the end of the world, but it does mean I have to wait until inside the method before I alias the logger now.